shell/browser/lib/shutdown_blocker_win.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2020 Microsoft, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+//
+// The purpose of this class is to block system shutdown on windows so the
+// application can exit cleanly. It uses the WM_QUERYENDSESSION message to do
+// so, as explained here
+// https://msdn.microsoft.com/en-us/library/windows/desktop/aa376890(v=vs.85).aspx
+//
+// The WM_QUERYENDSESSION message must be handled in every every process managed
```

Comment:
I'm not familiar with the `WM_QUERYENDSESSION` message, but if it must be handled in every process then we also need to handle it in GPU/utility/network/... processes then.